packages/kubernetes-1.26/kubernetes-1.26.spec

```diff
@@ -164,6 +164,7 @@ make hack/update-codegen.sh
 unset CC
 export KUBE_BUILD_PLATFORMS="linux/%{_cross_go_arch}"
 export %{kube_cc}
+export GO_MAJOR="1.22"
```

Comment:
This should be set up near `FORCE_HOST_GO` so that codegen happens with the same version of Go that we use to build kubelet.